pkg/server/plugin/noderesolver/aws/iid.go

```diff
@@ -87,32 +37,9 @@ func (p *IIDResolverPlugin) SetLogger(log hclog.Logger) {
 
 // Configure configures the IIDResolverPlugin
 func (p *IIDResolverPlugin) Configure(ctx context.Context, req *spi.ConfigureRequest) (*spi.ConfigureResponse, error) {
-	// Parse HCL config payload into config struct
-	config := new(caws.SessionConfig)
-	if err := hcl.Decode(config, req.Configuration); err != nil {
-		return nil, iidError.New("unable to decode configuration: %v", err)
-	}
-
-	// Set defaults from the environment
-	if config.AccessKeyID == "" {
-		config.AccessKeyID = p.hooks.getenv(caws.AccessKeyIDVarName)
-	}
-	if config.SecretAccessKey == "" {
-		config.SecretAccessKey = p.hooks.getenv(caws.SecretAccessKeyVarName)
-	}
-
-	switch {
-	case config.AccessKeyID != "" && config.SecretAccessKey == "":
-		return nil, iidError.New("configuration missing secret access key, but has access key id")
-	case config.AccessKeyID == "" && config.SecretAccessKey != "":
-		return nil, iidError.New("configuration missing access key id, but has secret access key")
-	}
-
-	// set the AWS configuration and reset clients
-	p.mu.Lock()
-	defer p.mu.Unlock()
-	p.config = config
-	p.clients = make(map[string]awsClient)
+	p.log.Warn("Usage of deprecated plugin detected.",
+		telemetry.PluginName, caws.PluginName,
+		telemetry.PluginType, noderesolver.Type)
```

Comment:
nit: I think we should get specific here and call them to action 
```suggestion
	p.log.Warn("The aws_iid resolver has been subsumed by the aws_iid node attestor and will be removed in a future release. Please remove it from your configuration.")
```

Reply:
Makes sense